### PR TITLE
"Standardise" docs and other items

### PR DESCRIPTION
**Pull request by xross**
- “Standardise” readme
- Parameterise XMOSDOC_VERISON and INFR_APPS_VERSION
- Fix lib_checks fail
- Add missing include (#72)
- Tidy up the JenkinsFile to more closely match other libs and recommended practice 
- Reference figs in documentation since they are now floating in the most recent xmosdoc
- (My editor removed a load of white space)

### CODE REVIEW

**Comment by xross** [comment]
@danielpieczko @shuchitak please check the python arrangement as discussed earlier. Thanks!

File: python/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+soundfile==0.12.1
```
Comment (danielpieczko) [inline python/requirements.txt:1]: I think this whole file should go because setup.py provides these requirements.

File: python/setup.py [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
     name="sw_pll",
-    version="2.0.0",
+    version="3.2.0",
     packages=["sw_pll"],
     package_dir={
-        "": "python"
+        "": "."
     },
     install_requires=[
         "numpy",
```
Comment (danielpieczko) [inline python/setup.py:13]: I think these should be pinned using the `~=` relation, maybe with major and minor, so `numpy~=2.1.*` ? Or maybe just major?

**Review by shuchitak — APPROVED** [review]
requirements and setup.py file placement looks fine to me. I agree with both of Daniel's comments about pinning dependency versions in setup.py and deleting tests/requirements.txt. Approving subject to fixing those.

File: python/requirements.txt (hunk 1 above)
Comment (xross) [reply python/requirements.txt]: We decided to move all deps to install_requires and make requirements just -e .

**Pull request merged**